.azuredevops/appcenter-sdk-unity-build-release-1es.yml

```diff
@@ -4,6 +4,19 @@ trigger:
 pr:
 - master
 - develop
+
+variables: 
+- name: EDITOR_URL_WIN
+  value: 'https://download.unity3d.com/download_unity/61fce66342ad/Windows64EditorInstaller/UnitySetup64-2018.4.18f1.exe'
+- name: IL2CPP_SUPPORT_URL
+  value: 'https://download.unity3d.com/download_unity/61fce66342ad/TargetSupportInstaller/UnitySetup-UWP-IL2CPP-Support-for-Editor-2018.4.18f1.exe'
+- name: NUGET_FEED_ID
+  value: 'AppCenter'
+- name: NUGET_USER
+  value: appcenter
+- name: AZURE_STORAGE_ACCOUNT
```

Comment:
Where is this variable used?

Reply:
It used in build.cake script, that running in PublishPackagesToStorage task. 
https://github.com/microsoft/appcenter-sdk-unity/blob/b85e7669d71a38c304646038ad26bc3819ca10c5/build.cake#L677

Reply:
I don't see this task in this pipeline. What do we push to the storage account?

Reply:
You are correct. Removed. Successful build: https://dev.azure.com/msmobilecenter/SDK/_build/results?buildId=1522401&view=logs&s=eb264492-b105-5667-1a96-783f82eb00ce

Reply:
What do we push to the storage account?